packages/makeswift-next-plugin/index.js

```diff
@@ -32,7 +32,7 @@ module.exports =
   ({
     resolveSymlinks,
     appOrigin = 'https://app.makeswift.com',
-    previewMode = true,
+    disableBuiltInPreview = false,
```

Comment:
Can we add a JSDoc comment explaining the options, including `disableBuiltInPreview`?

Reply:
Maybe something like `@param {boolean} disableBuiltInPreview - disables Makeswift's built-in preview mode. Set this only if you are providing a custom preview mode implementation.`?